documentation/certificates.md

```diff
@@ -0,0 +1,91 @@
+Streisand PKI
+=============
+
+Streisand includes a role responsible for PKI certificate generation,
+separated into two distinct tasks:
+
+ - CA/Server
+ - Client(s)
+
+The first of which will generate a certificate authority, and a server 
+certificate signed by the newly minted certificate authority.
+
+The latter task is responsible for generating client side certificate
+(using the same CA) used for authenticting connecting VPN clients.
+
+Each VPN service that leverages PKI generates its own CA/Server and 
+client certificates.
+
+Infrastructure
+--------------
+
+The overall PKI infrasture assumes the following layout:
+
+```
+Web Root CA: 
+   \__ gateway HTTPS cert (unless LE; see below)
+
+OpenConnect CA: 
+   \__ OpenConnect server cert
+   \__ OpenConnect client 1
+   \__ OpenConnect client ...
+   \__ OpenConnect client n
+
+OpenVPN CA: 
+   \__ OpenVPN server cert
+   \__ OpenVPN client 1
+   \__ OpenVPN client ...
+   \__ OpenVPN client n
+
+IPsec-IKEv2 CA (upcoming):
+   \__ IPsec-IKEv2 server cert
+   \__ foobar client 1
+   \__ foobar client ...
+   \__ foobar client n
+
+ISRG X1:
+   \__ Let's Encrypt Authority X3
+        \__ poem-walk.example.com HTTPS cert
+``` 
+
+Usage
+-----
+
+Utilizing the `certificates` role can be invoked as follows (using OpenVPN as an example):
+(Should client certificates need to be generated, they should be done so priort to invocation)
```

Comment:
typo: "priort".

I'm also not sure I understand the line about needing to generate client certs first. Can you clarify?